openfisca_core/legislations.py

```diff
@@ -30,9 +30,14 @@ def __init__(self, name, instant, variable_name = None):
         self.name = name
         self.instant = instant
         self.variable_name = variable_name
-        message = u'Legislation parameter "{}" was not found at instant "{}"'.format(name, instant)
+        message = u'\n\nThe parameter "{}"'.format(name)
```

Comment:
Inconsistency: you used simple quotes in most of the other examples.
Simple quotes are better in this case, as double quotes are not well rendered in the JSON:

```JSON
{
  "error": "Internal server error: The parameter \"taxes.income_tax_rat\" requested by variable \"income_tax\"  was not found in the 2017-04-01 tax and benefit system. If this code used to work and suddenly does not,it is most probably linked to a discrepancy in the Country Package code."
}
```